releasenotes.md

```diff
@@ -23,4 +23,30 @@ it is because one of the hostnames in `/etc/hosts` is longer than 63 characters.
 srun -N <number of nodes> sudo sed -i 's/\([0-9]\) .*pcluster / /' /etc/hosts
 ```
 
+* Relaunch a dynamic cluster created with `MinCount = 0` may fail due to compute nodes IP address mismatch.
 
+For dynamic cluster with `MinCount = 0`, /etc/hosts IP addresses of compute nodes may not match with what's in `nslookup` upon cluster relaunch. Therefore, for your information, a temporary workaround is included in `install_neuron.sh` post-install script:
```

Comment:
based on what I saw in the pipeline tests even without relaunch we noticed that this workaround was needed. the pipeline tests launch dynamic cluster and run jobs which then launches the workers and once job completes, we tear it down. It fails this use case without the workaround.

Reply:
ack